src/Controllers/Async.php

```diff
@@ -754,7 +754,7 @@ public function before(Request $request, Silex\Application $app)
 
         // If there's no active session, don't do anything.
         if (!$app['users']->isValidSession()) {
-            $app->abort(404, "You must be logged in to use this.");
+            $app->abort(Response::HTTP_NOT_FOUND, 'You must be logged in to use this.');
```

Comment:
One of the reasons I've changed these! It makes one look a bit closer :+1: 

But, yeah I'll update this one to HTTP_UNAUTHORIZED and yell if any others don't look right.